packages/maskbook/src/extension/background-script/EthereumServices/providers/MetaMask.ts

```diff
@@ -46,14 +39,6 @@ async function onError(error: string) {
     })
 }
 
-export async function updateIsMetaMaskLockedSettings() {
-    try {
-        currentIsMetaMaskLockedSettings.value = !(await provider?._metamask?.isUnlocked())
```

Comment:
Dose `await provider?._metamask?.isUnlocked()` not work fine?

Reply:
I think so.